docs/overview/changelog.md

```diff
@@ -132,6 +143,7 @@ _Released 2025-07-29_
 - Improved performance of maintaining full-text search greatly.
```

Comment:
I'm not quite sure how this line is supposed to read, but it's a little confusing as-is.